[2025-10-20T22:14:44.510+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-20T22:09:24.084565+00:00 [queued]>
[2025-10-20T22:14:44.531+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-20T22:09:24.084565+00:00 [queued]>
[2025-10-20T22:14:44.533+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-20T22:14:44.574+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_processor_harmonizer> on 2025-10-20 22:09:24.084565+00:00
[2025-10-20T22:14:44.586+0000] {standard_task_runner.py:60} INFO - Started process 929 to run task
[2025-10-20T22:14:44.593+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_scd2', 'run_processor_harmonizer', 'manual__2025-10-20T22:09:24.084565+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/process_scripts.py', '--cfg-path', '/tmp/tmp1cbs9mg8']
[2025-10-20T22:14:44.597+0000] {standard_task_runner.py:88} INFO - Job 37: Subtask run_processor_harmonizer
[2025-10-20T22:14:44.639+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-20T22:14:44.756+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_scd2.run_processor_harmonizer manual__2025-10-20T22:09:24.084565+00:00 [running]> on host 436d1e220711
[2025-10-20T22:14:44.943+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_scd2' AIRFLOW_CTX_TASK_ID='run_processor_harmonizer' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T22:09:24.084565+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T22:09:24.084565+00:00'
[2025-10-20T22:14:44.945+0000] {logging_mixin.py:188} INFO - ▶️ Exécution du script : /opt/***/scripts/processor_harmonizer.py
[2025-10-20T22:16:19.927+0000] {logging_mixin.py:188} INFO - ✅ 644/8723 lignes mises à jour, 3563 lignes insérées dans Extract_Station_20251020.xlsx
⚠ Ignoré (colonnes manquantes) : Inspections_HSE_20250815.xlsx
✅ 0/1997 lignes mises à jour, 815 lignes insérées dans Invariants_details_20251020.xlsx
✅ 0/1997 lignes mises à jour, 815 lignes insérées dans Invariants_study_20251020.xlsx
✅ 9/2281 lignes mises à jour, 0 lignes insérées dans Questions_HSE_20250815.xlsx
[2025-10-20T22:16:19.947+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-20T22:16:20.055+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_scd2, task_id=run_processor_harmonizer, execution_date=20251020T220924, start_date=20251020T221444, end_date=20251020T221620
[2025-10-20T22:16:20.206+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-20T22:16:20.375+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
